audit/types.go

```diff
@@ -144,160 +100,123 @@ type FormatterConfig struct {
 	RequiredFormat format
 }
 
-// EntryFilter should be used to filter audit requests and responses which should
-// make it to a sink.
-type EntryFilter struct {
-	// the evaluator for the bexpr expression that should be applied by the node.
-	evaluator *bexpr.Evaluator
-}
-
 // RequestEntry is the structure of a request audit log entry.
 type RequestEntry struct {
```

Comment:
Have you considered using the bexpr option WithTagName("json") so that we don't need to have these extra bexpr struct tags?

Reply:
There's also a similar mapstructure option, DecoderConfig.TagName.

Reply:
TIL! Amazing thanks for that Nick, will check it out and update.

Reply:
Wish we knew that before adding all of these 😆 Thanks!

<img src="https://media4.giphy.com/media/1L5YuA6wpKkNO/giphy.gif"/>